apps/healthlocker/web/templates/oxleas_adhd/caseload/user/interactions.html.eex

```diff
@@ -1,7 +1,7 @@
 <%= if @service_user == @user do %>
 <%= render "menu.html", assigns %>
   <div class="pv3 hl-dark-blue w-100 center flex justify-between">
-    <div class="w-60 pt6 pr3 br b--hl-dark-blue bw2 tc">
+    <div class="w-60 pt6 pr3 b--hl-dark-blue bw2 tc">
```

Comment:
if the border is removed we don't need then the other class which define the type of border:
- b--hl-dark-blue: the color of the boder (#242F4F)
- bw2: the border width size

So I think we can also remove these two classes